### PR TITLE
Update more chinese translate.

### PR DESCRIPTION
**Pull request by himenekocn**

### CODE REVIEW

File: translations/cs2kz-timer.phrases.txt [hunk 1]
```diff
@@ -16,25 +16,29 @@
...
 		"#format"	"player_name:s,course_name:s,time:s,mode:s,style:s"
 		"en"		"{lime}{player_name} {grey}finished the map in {default}{time}{grey}! [{purple}{mode}{grey} | {purple}{style}{grey} | {blue}PRO{grey}]"
+		"chi"		"{lime}{player_name} {grey}完成地图用时 {default}{time}{grey}! [{purple}{mode}{grey} | {purple}{style}{grey} | {blue}PRO{grey}]"
 	}
 	"Beat Course (NUB)"
 	{
 		// Bill finished course "Name" in 01:23.45 [Mode | Style | 12 TPs]
 		"#format"	"player_name:s,course_name:s,time:s,mode:s,style:s,tp_count_text:s"
-		"en"		"{lime}{player_name} {grey}finished course {bluegrey}\"{course_name}\" {grey}in {default}{time}{grey}! [{purple}{mode}{grey} | {purple}{style}{grey} | {tp_count_text}]"
+		"en"		"{lime}{player_name} {grey}finished course {bluegrey}\"{course_name}\" {grey}用时 {default}{time}{grey}! [{purple}{mode}{grey} | {purple}{style}{grey} | {tp_count_text}]"
```
Comment (zer0k-z) [inline translations/cs2kz-timer.phrases.txt:33]: This has chinese characters in it, probably copy paste mistake?

File: translations/cs2kz-timer.phrases.txt [hunk 2]
```diff
@@ -16,25 +16,29 @@
...
+		"chi"		"{lime}{player_name} {grey}完成关卡 {bluegrey}\"{course_name}\" {grey}用时 {default}{time}{grey}! [{purple}{mode}{grey} | {purple}{style}{grey} | {tp_count_text}]"
 	}
 	"Beat Course (PRO)"
 	{
 		// Bill finished course "Name" in 01:23.45 [Mode | Style | PRO]
 		"#format"	"player_name:s,course_name:s,time:s,mode:s,style:s"
-		"en"		"{lime}{player_name} {grey}finished course {bluegrey}\"{course_name}\" {grey}in {default}{time}{grey}! [{purple}{mode}{grey} | {purple}{style}{grey} | {blue}PRO{grey}]"
+		"en"		"{lime}{player_name} {grey}finished course {bluegrey}\"{course_name}\" {grey}用时 {default}{time}{grey}! [{purple}{mode}{grey} | {purple}{style}{grey} | {blue}PRO{grey}]"
```
Comment (zer0k-z) [inline translations/cs2kz-timer.phrases.txt:40]: Same as above

File: translations/cs2kz-styles.phrases.txt [hunk 3]
```diff
@@ -3,22 +3,26 @@
...
 		"en"		"Possible styles: (Current style is {style})"
+		"chi"		"可用风格: (当前风格为 {style})"
 	}
 	"Style Not Available"
 	{
 		// The AutoBhop style is currently not available.
 		"#format"	"style:s"
-		"en"		"{grey}The {purple}{style} {grey}style is currently not available."
+		"en"		"{grey}风格 {purple}{style} {grey}style is currently not available."
+		"chi"		"{grey}The {purple}{style} {grey}当前不可用."
```
Comment (zer0k-z) [inline translations/cs2kz-styles.phrases.txt:19]: Chinese characters in english and english characters in chinese

**Review by zer0k-z — CHANGES_REQUESTED** [review]

File: translations/cs2kz-styles.phrases.txt (hunk 3 above)
Comment (himenekocn) [reply translations/cs2kz-styles.phrases.txt:19]: HHHHHHHHH, my worng

**Review by zer0k-z — APPROVED** [review]

**Pull request merged**